app/models/concerns/orchestration/dhcp.rb

```diff
@@ -174,6 +175,18 @@ def ip_belongs_to_subnet?
     # we let other validations handle that
   end
 
+  def ip6_belongs_to_subnet6?
```

Comment:
This code is duplicated from ipv4, please refactor in a way there's no duplicate code. Also it's using `or` incorrectly, it should be `||` here, this is boolean "or", not control flow "or". Think about how different 

`a = false or true` is vs  `a = false || true`